01_submitting_contract/hello_world.etch

```diff
@@ -31,10 +31,10 @@ function main()
 
   printLn(greeting);
 endfunction
-*/
+
 
 @init
-function createMessage(owner: Address)
+function createMessage(s : String)
```

Comment:
This needs to be an `Address` - otherwise the it does not work with the ledger.

Reply:
oh, interesting, it errors on this